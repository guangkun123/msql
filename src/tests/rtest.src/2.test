# Insert some sample data into the tables

insert into test(user,age,phone)values( 'Bambi',26,'(075) 951450')\p\g
insert into test(user,age,phone)values( 'paulp',25,'(075) 951412')\p\g
insert into test(user,age,phone)values( 'fred',31,'(075) 951431')\p\g

insert into test(user,age,phone)values( 'dummy1',99,'99')\p\g
insert into test(user,age,phone)values( 'dummy2',97,'97')\p\g
insert into test(user,age,phone)values( 'dummy3',95,'95')\p\g
insert into test(user,age,phone)values( 'dummy4',93,'93')\p\g
insert into test(user,age,phone)values( 'dummy5',91,'91')\p\g
insert into test(user,age,phone)values( 'dummy6',89,'89')\p\g
insert into test(user,age,phone)values( 'dummy7',87,'87')\p\g

insert into test(user,age,phone)values( 'fred',31,'(075) 951431')\p\g
insert into test(user,age,phone)values( NULL,31,'(075) 951431')\p\g

insert into test2 (user, fullname) values ('paulp','Paul Pyyvaara') \p\g
insert into test2 (user, fullname) values ('Bambi','David J. Hughes') \p\g
insert into test2 (user, fullname) values ('fred','Fred O\'Niell') \p\g
insert into test2 (user, fullname) values (NULL,'Nobody') \p\g
insert into test2 (user, fullname) values ('Nobody',NULL) \p\g


