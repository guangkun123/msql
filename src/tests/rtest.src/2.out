
Welcome to the miniSQL monitor.  Type \h for help.


mSQL >     -> 
Query buffer
------------

insert into test(user,age,phone)values( 'Bambi',26,'(075) 951450')
[continue]
    -> 
Query OK.




mSQL > 
Query buffer
------------
insert into test(user,age,phone)values( 'paulp',25,'(075) 951412')
[continue]
    -> 
Query OK.




mSQL > 
Query buffer
------------
insert into test(user,age,phone)values( 'fred',31,'(075) 951431')
[continue]
    -> 
Query OK.




mSQL >     -> 
Query buffer
------------
insert into test(user,age,phone)values( 'dummy1',99,'99')
[continue]
    -> 
Query OK.




mSQL > 
Query buffer
------------
insert into test(user,age,phone)values( 'dummy2',97,'97')
[continue]
    -> 
Query OK.




mSQL > 
Query buffer
------------
insert into test(user,age,phone)values( 'dummy3',95,'95')
[continue]
    -> 
Query OK.




mSQL > 
Query buffer
------------
insert into test(user,age,phone)values( 'dummy4',93,'93')
[continue]
    -> 
Query OK.




mSQL > 
Query buffer
------------
insert into test(user,age,phone)values( 'dummy5',91,'91')
[continue]
    -> 
Query OK.




mSQL > 
Query buffer
------------
insert into test(user,age,phone)values( 'dummy6',89,'89')
[continue]
    -> 
Query OK.




mSQL > 
Query buffer
------------
insert into test(user,age,phone)values( 'dummy7',87,'87')
[continue]
    -> 
Query OK.




mSQL >     -> 
Query buffer
------------
insert into test(user,age,phone)values( 'fred',31,'(075) 951431')
[continue]
    -> 

ERROR : Non unique key value in field 'user'


mSQL > 
Query buffer
------------
insert into test(user,age,phone)values( NULL,31,'(075) 951431')
[continue]
    -> 

ERROR : Field "user" cannot be null


mSQL >     -> 
Query buffer
------------
insert into test2 (user, fullname) values ('paulp','Paul Pyyvaara') 
[continue]
    -> 
Query OK.




mSQL > 
Query buffer
------------
insert into test2 (user, fullname) values ('Bambi','David J. Hughes') 
[continue]
    -> 
Query OK.




mSQL > 
Query buffer
------------
insert into test2 (user, fullname) values ('fred','Fred O\'Niell') 
[continue]
    -> 
Query OK.




mSQL > 
Query buffer
------------
insert into test2 (user, fullname) values (NULL,'Nobody') 
[continue]
    -> 

ERROR : Field "user" cannot be null


mSQL > 
Query buffer
------------
insert into test2 (user, fullname) values ('Nobody',NULL) 
[continue]
    -> 
Query OK.




mSQL >     ->     -> 
Bye!

